R/tbl_summary.R

```diff
@@ -91,6 +91,10 @@
 #'   \item `{foo}` any function of the form `foo(x)` is accepted where `x` is a numeric vector
 #' }
 #'
+#' For both categorical and continuous variables, statistics on the number of
+#' missing and non-missing observations and their proportions are available to print.
+#' `{N_obs}`, `{N_miss}`, `{N_nonmiss}`, `{p_miss}`, `{p_nonmiss}`
```

Comment:
To be consistent with the above, can this be written as:
For both categorical and continuous variables, statistics on the number of missing and non-missing observations and their proportions are available to display. 
* {N_obs} total number of observations
* {N_miss} number of missing observations
* {N_nonmiss} number of non-missing observations
* {p_miss} percentage of observations missing
* {p_nomiss} percentage of observations not missing

It may also be worth including a brief sentence to clarify that statistic names starting with capital N refer to the denominator for categorical variables. Something like "Note that for categorical variables, N_obs, N_miss and N_nonmiss refer to the total number, number missing and number non missing observations in the denominator, not at each level of the categorical variable. 

Also, for categorical variables, is there a difference between N_obs and N? For consistency, it may be worth using N throughout since that already exists.